spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/RibbonCommand.java

```diff
@@ -80,7 +80,7 @@ public RibbonCommand(String commandKey, RestClient restClient, Verb verb, String
 		super(getSetter(commandKey));
 		this.restClient = restClient;
 		this.verb = verb;
-		this.uri = (StringUtils.hasText(uri))? UriComponentsBuilder.fromUriString(uri).build().toUri() : new URI(uri);
```

Comment:
My bad, I was thinking the exceptional case was with `URI` if `uri` was empty, but it was with `UriComponentsBuilder`.